components/resourceDetails/resourceSharing/agentAccessTable/styles.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { createStyles } from "@solid/lit-prism-patterns";
+
+export default function styles(theme) {
+  return createStyles(theme, ["icons", "table"], {
+    accordion: {
+      padding: theme.spacing(1.8, 1.6),
+      color: theme.palette.secondary.contrastText,
+    },
+    rounded: {
+      borderRadius: "4px",
+      boxShadow: `0 1px 4px 0 ${theme.palette.grey[500]}`,
+    },
+    title: {
+      fontSize: theme.typography.h4.fontSize,
+      fontWeight: 800,
+    },
+    headerContainer: {
+      display: "flex",
+      flexDirection: "row",
+      alignItems: "center",
+      marginBottom: theme.spacing(1.6),
+      "&& p": {
+        padding: 0,
+        margin: 0,
+      },
+    },
+    textContainer: {
+      width: "100%",
+      flexDirection: "column",
+    },
+    description: {
+      fontSize: "12px",
```

Comment:
Again, think this can be nudged to 13px, to keep the small font-sizes consistent?